src/python/pants/backend/jvm/tasks/coverage/scoverage.py

```diff
@@ -86,6 +97,7 @@ def __init__(self, report_path, target_filters, settings, targets, execute_java_
                                      should also accept `*args` and `**kwargs` compatible with the
                                      remaining parameters accepted by
                                      `pants.java.util.execute_java`.
+    :param str coverage_output_dir: An optional output directory to copy coverage reports to.
```

Comment:
Do we need this to be configurable, or is copying into `dist` sufficient?

Reply:
Copying into `dist/` should be great!